### PR TITLE
Joke hint adjustments

### PR DESCRIPTION
**Pull request by SpaghettiToastBook**
Fixes #403.

Unfortunately the "Metroids are not pets" joke hint is a little bit too long to fit in the text box. We could make the hint go over multiple pages, but I don't know if it's worth bothering to do for just that.

### CODE REVIEW

File: randovania/games/prime/patcher_file_lib/item_hints.py [hunk 1]
```diff
@@ -43,15 +43,13 @@
 ]
 
 _JOKE_HINTS = [
```
Comment (henriquegemignani) [inline randovania/games/prime/patcher_file_lib/item_hints.py]: We should leave the rules for a joke hint to be added as a comment here.

**Comment by henriquegemignani** [comment]
We need more joke hints :(

File: randovania/games/prime/patcher_file_lib/item_hints.py (hunk 1 above)
Comment (SpaghettiToastBook) [reply randovania/games/prime/patcher_file_lib/item_hints.py]: Agreed. Are the ones I put in #403 okay?

Comment (henriquegemignani) [reply randovania/games/prime/patcher_file_lib/item_hints.py]: I think you meant the issue, not a circular link. But yeah.

**Pull request merged**